components/Base/Button/index.vue

```diff
@@ -72,6 +76,9 @@ export default {
         return props
       }
       return this.$props
+    },
+    renderClasses () {
+      return `jds-button--${this.variant}-${this.size === 'base' ? '' : this.size}`
```

Comment:
hello @adzharamrullah , if the size property has a value of `base`, the `renderClasses` would be :
`jds-button--primary-` isn't it? I think the `-` at the end isn't necessary 🤔

Reply:
Thanks, i have fixed that